tests/_support/AcceptanceTester.php

```diff
@@ -472,4 +472,18 @@ public function checkEmailWasReceived($subject) {
       $i->waitForText($subject, 60);
     }
   }
+
+  /**
+   * Makes sure that there is a newsletter template of given order on given template tab
+   * @return string Template element selector
+   */
+  public function checkTemplateIsPresent(int $templateIndex, string $templateCategory = 'standard'): string {
+    $templateTab = "[data-automation-id=\"templates-$templateCategory\"]";
```

Comment:
Just for my information, you don't have to change the code now. Why didn't you use this notation? 
```
    $templateTab = "[data-automation-id='templates-$templateCategory']";
```
It seems cleaner to me. And afaik it works the same

Reply:
Ahh, I thought that the selector must contain double-quotes. Good to know. Next time I'll use single quotes without escaping.